src/util/fields.py

```diff
@@ -654,6 +654,23 @@ def convert_field_to_html(cr, model, field, skip_inherit=()):
         """,
         [field, model],
     )
+
+    # Update ir.default
+    if table_exists(cr, "ir_default"):
+        cr.execute(
+            r"""
+            UPDATE ir_default AS d
+               SET json_value = REPLACE(
+                                    REPLACE(json_value, '\n', '<br/>'),
+                                    '\t', '&nbsp;&nbsp;&nbsp;&nbsp;'
```

Comment:
This should be part of `pg_text2html`.
Instead of 4 spaces, we can use the `&emsp;` entity (em space, [U+2003](https://unicode-explorer.com/c/2003))

Reply:
> we can use the `&emsp;` entity

Or just `&Tab;`.